content/blog/2022/03/admission-policy.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Introducing the AdmissionPolicy
+authors:
+- Raul Cabello Martin
+date: 2022-03-09
+---
+
+Up until now, the only way to define a policy in Kubewarden was to use a `ClusterAdmissionPolicy`. `ClusterAdmissionPolicies` are applied to matching cluster-wide resources and resources across all namespaces.
+
+We are thrilled to announce `AdmissionPolicy`, a namespaced resource. The policy will process only the requests that are targeting the namespace where the `AdmissionPolicy` is defined. Other than that, there are no functional differences between `AdmissionPolicy` and `ClusterAdmissionPolicy`.
+
+## Understanding the motivation for using AdmissionPolicies
+
+It is possible to restrict the namespaces where a `ClusterAdmissionPolicy` evaluates resources using a `namespaceSelector`, but there is no way for kubernetes administrators to restrict users from creating `ClusterAdmissionPolicies` that evaluate resources just in a particular namespace. 
```

Comment:
Nit (and I'm not sure if that's correct even)

```suggestion
It is possible to restrict the namespaces where a `ClusterAdmissionPolicy` evaluates resources using a `namespaceSelector` (in K8s >= 1.21), but there is no way for Kubernetes administrators to restrict users from creating `ClusterAdmissionPolicies` that evaluate resources in any namespace, once they are allowed to create them.
```

Reply:
`kubernetes.io/metadata.name` label, which is the namespace name, was introduced in k8s 1.21. I think you can still apply other filters with the namespaceSelector in versions below 1.21

Reply:
Ook, resolving then.